brunch-config.js

```diff
@@ -28,5 +34,14 @@ module.exports = {
         })
       ]
     }
+  },
+  sourceMaps: 'inline', // For karma-sourcemap-loader.
+  overrides: {
+    production: {
+      paths: {
+        // Exclude files in "test" from compilation.
+        watched: ['app', 'vendor']
```

Comment:
As far as I can tell, the `vendor` directory does not exist within the project (nor is it ever generated). Did you mean to add this?